### PR TITLE
add bytes size output in block and transaction

### PR DESCRIPTION
**Pull request by ShiningRay**
Add "size" field in block outputs in API 
Add "bytes" field in transaction outputs in API

### CODE REVIEW

**Comment by Keith-CY** [comment]
Please add a brief description of this PR and add a link of the feature request issue

**Comment by Keith-CY** [comment]
CI failed

**Comment by Keith-CY** [comment]
Why name sizes of block and transaction in different names?

File: app/workers/generate_statistics_data_worker.rb [hunk 1]
```diff
@@ -17,6 +17,10 @@ def perform(block_id)
     end
 
     CellOutput.upsert_all(cell_outputs_attributes) if cell_outputs_attributes.present?
-
+    block.ckb_transactions.find_each do |c|
+      res = CkbSync::Api.instance.directly_single_call_rpc(method: :get_transaction, params: [c.tx_hash, '0x0'])
+      c.bytes = (res['result']['transaction'].size - 2)/2
```
Comment (quake) [inline app/workers/generate_statistics_data_worker.rb:22]: should use transaction serialized data size directly, no need to call rpc

**Comment by ShiningRay** [comment]
This PR should be merged until new CKB Node version 0.105 is deployed

File: app/workers/generate_statistics_data_worker.rb (hunk 1 above)
Comment (ShiningRay) [reply app/workers/generate_statistics_data_worker.rb:22]: use `Transaction#serialized_size_in_block` ?

Comment (ShiningRay) [reply app/workers/generate_statistics_data_worker.rb:22]: As you see, this is a background worker for update such information later.
And the former programmer also send request to RPC and allocate new Block object here:

https://github.com/nervosnetwork/ckb-explorer/blob/develop/app/workers/generate_statistics_data_worker.rb#L8

**Review by iamyates — APPROVED** [review]

File: app/workers/generate_statistics_data_worker.rb (hunk 1 above)
Comment (quake) [reply app/workers/generate_statistics_data_worker.rb:22]: > use Transaction#serialized_size_in_block ?
Yes,

> And the former programmer also send request to RPC and allocate new Block object here:


I think this rpc call can be omitted also by constructing from the hashmap:
```
node_block = CKB::Types::Block::from_h(block.to_raw)
```

Comment (Keith-CY) [reply app/workers/generate_statistics_data_worker.rb:22]: @ShiningRay will we refactor data fetching in this PR, or open another issue for it.

**Pull request merged**